pkg/cmd/pulumi/about.go

```diff
@@ -423,7 +427,12 @@ func (current currentStackAbout) String() string {
 			Rows:    rows,
 		}.String() + "\n"
 	}
-	return fmt.Sprintf("Current Stack: %s\n\n%s\n%s", current.Name, resources, pending)
+	return fmt.Sprintf(`Current Stack: %s
+
+Fully qualified stack name: %s
```

Comment:
To help support filestate, but _also_ to support httpstate when the current defaults don't match the current stack (and so ref.String() has printed fully qualified anyway) only print the "Fully qualified stack name: %s" iff its different to `current.Name`